src/css/custom.css

```diff
@@ -43,6 +43,20 @@ code, .prism-code {
   font-family: ui-monospace,SFMono-Regular,SF Mono,Menlo,Consolas,Liberation Mono,monospace;
   background-color: #F6F8FA;
   border: none;
+  counter-reset: line-number;
+}
+
+/* add number line */
+.prism-code.language-javascript .token-line::before {
+  counter-increment: line-number;
+  content: counter(line-number);
+  margin-right: calc(var(--ifm-pre-padding) * 1.5);
+  text-align: right;
+  min-width: 2.5rem;
```

Comment:
I really like the numbering!

I was wondering if we could reduce the width of the line number to save some space by changing this value to `1.5rem` which seems to work up to three digits.

```suggestion
  min-width: 1.5rem;
```

## After
<img width="910" alt="Screen Shot 2021-11-05 at 3 15 21 PM" src="https://user-images.githubusercontent.com/9922882/140566177-827f49d5-f0e6-4303-9bbd-5059d25aeb18.png">

## Before 
<img width="919" alt="Screen Shot 2021-11-05 at 3 15 57 PM" src="https://user-images.githubusercontent.com/9922882/140566241-496b93b4-7f56-4185-9406-b13341dd0eed.png">

Reply:
sounds great to me!